ts/Extensions/Breadcrumbs.ts

```diff
@@ -0,0 +1,658 @@
+                    format(breadcrumbsOptions.format as any,
+                        { value: breadcrumb[1].name || lang?.mainBreadCrumb }, chart),
+                posX,
+                posY,
+                function (): void {
+                    if (breadcrumbsOptions.events && breadcrumbsOptions.events.click) {
+                        breadcrumbsOptions.events.click(button as any, breadcrumbs as any);
+                        // Prevent from click on the current level
+                    } else if (breadcrumbsList[breadcrumbsList.length - 1][1].name !== button.textStr) {
+                        breadcrumbs.multipleDrillUp(breadcrumb[0]);
+                    }
+                })
+                .attr({
+                    padding: 3
+                })
+                .css(breadcrumbsOptions.style as any)
+                .addClass('highcharts-drilldown-breadcrumbs-button')
+                .add(breadcrumbs.breadcrumbsGroup);
+            return button;
+        }
+    }
+
+    /**
+    * Render the separator.
+    *
+    * @requires  modules/drilldown
+    *
+    * @function Highcharts.Chart#renderSeparator
+    * @param {Highcharts.Breadcrumbs} this
+    *        Breadcrumbs class.
+    * @param {Highcharts.Breadcrumbs} breadcrumb
+    *        Current breadcrumb
+    * @param {Highcharts.Breadcrumbs} posX
+    *        Initial horizontal position
+    * @param {Highcharts.Breadcrumbs} posY
+    *        Initial vertical position
+    * @return {SVGElement|void}
+    *        Returns the SVG button
+    */
+    public renderSeparator(this: Breadcrumbs, breadcrumb: Array<any>, posX: number, posY: number): SVGElement|void {
+        const breadcrumbs = this,
+            chart = this.chart,
+            breadcrumbsOptions = chart.options.drilldown?.breadcrumbs;
+
+        if (breadcrumbsOptions) {
+            return chart.renderer.symbol(
+                breadcrumbsOptions.separator as string,
+                posX + 5,
+                posY - 10,
+                10,
+                10
+            )
+                .attr({
+                    fill: 'white',
+                    stroke: 'black',
+                    'stroke-width': 1,
+                    rotation: 90,
+                    rotationOriginX: posX,
+                    rotationOriginY: posY
+                })
+                .add(breadcrumbs.breadcrumbsGroup)
+                .addClass('highcharts-drilldown-breadcrumbs-separator');
+        }
+    }
+
+    /**
+    * Update.
+    *
+    * @requires  modules/drilldown
+    *
+    * @function Highcharts.Chart#update
+    * @param {Highcharts.Breadcrumbs} this
+    *        Breadcrumbs class.
+    * @param {Highcharts.Options} userOptions
+    *        Breadcrumbs class.
+    */
+    public update(this: Breadcrumbs, userOptions: Highcharts.Options): void {
+        const chartOptions = H.merge(userOptions, this.options);
```

Comment:
We are updating the options but not the buttons themselves. I think we should draw buttons again in here (or in `redraw` method). If in `redraw`, we should add `redraw` flag as `update` method's argument.